ddl.go

```diff
@@ -169,10 +169,15 @@ func dropIndex() (string, string) {
 		b         strings.Builder
 	)
 
+	timeout := 100
```

Comment:
100 might be too small, it only wait for 10s at most.
BTW, I think `retryLimit` is a better name, it's a counter, not a duration.